### PR TITLE
HTML5.LinkTarget

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

File: vocabularies/HTML5.xml [hunk 1]
```diff
@@ -49,6 +49,16 @@
         </Property>
       </ComplexType>
 
+      <Term Name="LinkTarget" Type="Edm.String" Nullable="true">
```
Comment (ralfhandl) [inline vocabularies/HTML5.xml:52]: How about making this a type definition with AllowedValues, describing (a subset of) the generic link targets `_self`, `_blank`, `_parent`, and `_top`?

**Review by ralfhandl — CHANGES_REQUESTED** [review]

File: vocabularies/HTML5.xml (hunk 1 above)
Comment (HeikoTheissen) [reply vocabularies/HTML5.xml:52]: This is the HTML5 vocabulary, why not use the HTML5 terms here?

**Review by ralfhandl — CHANGES_REQUESTED** [review]

File: vocabularies/HTML5.json [hunk 2]
```diff
@@ -36,22 +39,18 @@
...
     },
     "LinkTargetType": {
       "$Kind": "TypeDefinition",
       "$UnderlyingType": "Edm.String",
+      "@Core.Description": "The values are interpreted like the [target attribute](https://html.spec.whatwg.org/multipage/links.html#attr-hyperlink-target) in HTML",
       "@Validation.AllowedValues": [
-        { "Value": "_self", "@Core.Description": "Open link in the current browsing context" },
-        { "Value": "_blank", "@Core.Description": "Open link in a new browsing context" },
-        { "Value": "_parent", "@Core.Description": "Open link in the parent browsing context" },
-        { "Value": "_top", "@Core.Description": "Open link in the top browsing context" }
```
Comment (ralfhandl) [inline vocabularies/HTML5.json]: I liked these descriptions, the table of allowed values looks empty without them.

File: vocabularies/UI.xml [hunk 3]
```diff
@@ -1018,6 +1023,11 @@ The trend is
         <Property Name="Url" Type="Edm.String" Nullable="false">
           <Annotation Term="Core.IsURL" />
           <Annotation Term="Core.Description" String="URL of referenced information" />
+          <Annotation Term="Validation.ApplicableTerms">
```
Comment (BerSie) [inline vocabularies/UI.xml:1026]: what should happen to the facet if you annotate it with "LinkTarget" - how should the UI behave?

Comment (HeikoTheissen) [reply vocabularies/UI.xml:1026]: When the UI contains a link to that URL, the link shall open according to the `HTML5.LinkTarget`. In other words: The `sap.m.Link` element shall get a corresponding `target` property.

**Review by ralfhandl — APPROVED** [review]

**Review by BerSie — APPROVED** [review]

**Pull request merged**